securedrop/tests/conftest.py

```diff
@@ -91,14 +89,11 @@ def setup_journalist_key_and_gpg_folder() -> Generator[Tuple[str, Path], None, N
         gpg = gnupg.GPG("gpg2", homedir=str(tmp_gpg_dir))
         journalist_public_key_path = Path(__file__).parent / "files" / "test_journalist_key.pub"
         journalist_public_key = journalist_public_key_path.read_text()
-        journalist_key_fingerprint = gpg.import_keys(journalist_public_key).fingerprints[0]
+        # TODO: we don't need the journalist pub key in the keyring anymore, but include
+        # it anyways to match a legacy prod keyring.
+        gpg.import_keys(journalist_public_key)
 
-        # Reduce source GPG key length to speed up tests at the expense of security
-        with mock.patch.object(
-            EncryptionManager, "GPG_KEY_LENGTH", PropertyMock(return_value=1024)
-        ):
-
-            yield journalist_key_fingerprint, tmp_gpg_dir
+        yield "65A1B5FF195B56353CC63DFFCC40EF1228271441", tmp_gpg_dir
```

Comment:
Nit: Why hard-code this rather than print the fingerprint of the key newly imported into the keyring?

Reply:
IIRC I was debugging something that wasn't working and hardcoded it to avoid one potential issue and didn't go back and undo it. Will revert it.